service/debugger/debugger.go

```diff
@@ -1145,7 +1145,7 @@ func (d *Debugger) convertDefers(defers []*proc.Defer) []api.Defer {
 }
 
 // FindLocation will find the location specified by 'locStr'.
-func (d *Debugger) FindLocation(scope api.EvalScope, locStr string) ([]api.Location, error) {
+func (d *Debugger) FindLocation(scope api.EvalScope, locStr string, returnLines bool) ([]api.Location, error) {
```

Comment:
We should be consistent here and name `returnLines` to `findInstruction`, it also seems more clear to me.

Reply:
To keep backwards compatibility the default value of the API field (false) must produce the old behavior, hence the switch between findInstruction and returnLines. We could use findInstruction here but the member field that the server API takes needs to stay returnLines (or another name with the same semantics)

Reply:
Ack, good call. How about `includeNonExecutableLines` or something to that effect? I just want to make it a little clearer what is going on and have the name be a bit closer to what we're using elsewhere.

Don't mean to get too dragged down on bikeshedding.